### PR TITLE
Updated docs for 0.16 release

### PR DESCRIPTION
**Pull request by radinamatic**
## Summary

I think this should be all, a part from the dev docs & release notes:

- http://ka-lite.readthedocs.org/en/0.16.x/developer_docs/index.html
- http://ka-lite.readthedocs.org/en/0.16.x/installguide/release_notes.html

@aronasorman, sorry for the delay!

@cpauya I think I got all for the Mac parts too, but you better check once @aronasorman merges it!

### CODE REVIEW

**Comment by radinamatic** [comment]
And while you're at it tomorrow, check the Advanced installation too, I think nothing changed there, but it is, well... advanced. :wink: 

http://ka-lite.readthedocs.org/en/0.16.x/installguide/advanced.html

@aronasorman

File: docs/installguide/install_all.rst [hunk 1]
```diff
@@ -31,16 +31,13 @@ or use the command-line ``kalite`` program, which in typical installations can b
...
-#. Click the ``Agree`` button to accept the LICENSE agreement.
-#. On the .dmg window, drag the ``KA-Lite Monitor`` app into the ``Applications`` folder.
-#. Launch ``KA-Lite Monitor`` from your ``Applications`` folder.
+#. Download the KA Lite `OSX installer <https://learningequality.org/r/osx-installer-0-16>`_.
+#. After the download is complete, double click the .pkg file.
+#. Click the ``Agree`` button to accept the LICENSE agreement and follow the installer instructions.
+#. Launch ``KA-Lite app`` from your ``Applications`` folder.
 #. On first load, it will check your current environment and show the Preferences dialog.
 #. Input your preferred administrator username and password, then click the ``Apply`` button in ``KA-Lite Preferences`` dialog.
 #. You will be prompted that initial setup will take a few minutes, click the ``OK`` button and wait for the notification that KA-Lite has been setup and can now be started.
```
Comment (cpauya) [inline docs/installguide/install_all.rst]: The KA Lite app is automatically launched during installation and we no longer ask for the admin account.  Please remove the steps from "Click the Agree button..." until "You will be prompted...".

Let's replace those 4 steps with these:
* Click on the Continue button to allow the installer program to check for pre-installation requirements.
* Follow the prompts in the installer dialog to install KA Lite.
* The "KA Lite app" will be launched automatically during installation, and should show notifications and a menu bar icon.
* When the installation is done, you will be notified that "KA Lite is running...".  The installer should also show the "Summary" page with instructions to start using KA Lite.
* To start using KA Lite, click on the menu bar icon and select "Open in Browser".

File: docs/installguide/install_all.rst [hunk 2]
```diff
@@ -54,23 +51,11 @@ _______
...
-    #. Click on ``Preferences`` in the menu option.
-    #. Go to ``Advanced`` tab, click on the ``Extract Assessment`` button, then confirm the action. 
-    #. Restart the server, login to the web app, then check the sidebar contents.
-
-    This will enable the greyed-out assessment items on the sidebar.
+#. Download the KA Lite `OSX installer <https://learningequality.org/r/osx-installer-0-16>`_.
+#. Make sure that you stop the server and quit the KA Lite Monitor.
+#. After the download is complete, double click the .pkg file.
+#. Click the ``Agree`` button to accept the LICENSE agreement and follow the installer instructions.
+#. Launch ``KA-Lite app`` from your ``Applications`` folder.
```
Comment (cpauya) [inline docs/installguide/install_all.rst]: I think the same changes applies as per my comment above.  Let's replace the "Click the Agree button" and the "Launch KA-Lite app" steps with the steps I outlined above.

**Comment by radinamatic** [comment]
@aronasorman Pushed a couple of changes by Liz & @cpauya, CI is complaining but I can't see anything related to docs...

File: docs/faq.rst [hunk 3]
```diff
@@ -76,23 +84,9 @@ To install, a language pack on a computer without internet access, look up the l
...
-  kalite manage languagepackdownload --from-file=<path to zip file> -l<language code> --commandline
-
-Or for Windows::
-
-  <path to KA Lite installation>\bin\windows\kalite.bat languagepackdownload --from-file=<path to zip file> -l<language code> --commandline
-
 An example invocation for Windows would be::
 
-  C:\Program Files\KA Lite\ka-lite\bin\windows\kalite.bat languagepackdownload --from-file="C:\Downloads\es.zip" -les --commandline
+    C:\Program Files\KA Lite\ka-lite\bin\windows\kalite.bat kalite manage etrievecontentpack local en en.zip
```
Comment (aronasorman) [inline docs/faq.rst]: Typo: `eretrievecontentpack`.

File: docs/faq.rst [hunk 4]
```diff
@@ -228,17 +222,14 @@ Note that online data back-ups occur if you "register" your KA Lite installation
...
 ----------------------------------------------------------------------------------------
 
 KA Lite on Windows is controlled through a task-tray program.
 See the :doc:`installation guide <installguide/install_all>` for some more info.
+
+
+I can't see videos in Firefox on Ubuntu/Debian!
+-----------------------------------------------
+
+Install `Ubuntu restricted extras package <https://apps.ubuntu.com/cat/applications/ubuntu-restricted-extras/>`_ in Ubuntu Software Center.
```
Comment (aronasorman) [inline docs/faq.rst]: `in the Ubuntu Software Center`.

File: docs/installguide/install_all.rst [hunk 5]
```diff
@@ -31,46 +31,31 @@ or use the command-line ``kalite`` program, which in typical installations can b
...
-#. You will be prompted that initial setup will take a few minutes, click the ``OK`` button and wait for the notification that KA-Lite has been setup and can now be started.
-#. Click on the KA-Lite logo icon on the menu bar and select the ``Start KA-Lite`` menu option.
-#. Wait for the notification that you can now click on ``Open in Browser`` menu option.
-#. Click on the KA-Lite logo icon on the menu bar and select ``Open in Browser`` menu option - this should launch KA-Lite in your preferred web browser.
-#. Login using the administrator account you have specified during setup.
+#. Download the KA Lite `OSX installer <https://learningequality.org/r/osx-installer-0-16>`_.
+#. After the download is complete, double click the .pkg file.
+#. Click on the Continue button to allow the installer program to check for pre-installation requirements.
+#. Follow the prompts in the installer dialog to install KA Lite.
+#. The "KA Lite app" will be launched automatically during installation, display notifications and a menu bar icon.
```
Comment (aronasorman) [inline docs/installguide/install_all.rst]: > The "KA Lite app" will launch automatically during installation, display both notifications and a menu bar icon.

File: docs/installguide/install_all.rst [hunk 6]
```diff
@@ -31,46 +31,31 @@ or use the command-line ``kalite`` program, which in typical installations can b
...
-    #. Go to ``Advanced`` tab, click on the ``Extract Assessment`` button, then confirm the action. 
-    #. Restart the server, login to the web app, then check the sidebar contents.
-
-    This will enable the greyed-out assessment items on the sidebar.
+#. Download the KA Lite `OSX installer <https://learningequality.org/r/osx-installer-0-16>`_.
+#. Make sure that you stop the server and quit the KA Lite Monitor.
+#. After the download is complete, double click the .pkg file.
+#. Click on the Continue button to allow the installer program to check for pre-installation requirements.
+#. Follow the prompts in the installer dialog to install KA Lite.
+#. The "KA Lite app" will be launched automatically during installation, display notifications and a menu bar icon.
```
Comment (aronasorman) [inline docs/installguide/install_all.rst]: > The "KA Lite app" will launch automatically during installation, display both notifications and a menu bar icon.

File: docs/usermanual/userman_admin.rst [hunk 7]
```diff
@@ -29,57 +29,59 @@ If you choose to register online, these important terms will help you better und
...
 Setting up KA Lite
 ------------------
-Once you have successfully installed KA Lite, the installation script should give you a URL (http://127.0.0.1:8008/) to visit so that you can open KA Lite and login for the first time.
+Once you have successfully installed KA Lite, the installation script will give you a URL (http://127.0.0.1:8008/) to visit so that you can open KA Lite and login for the first time.
 
-Copy and paste the URL into a web browser. The KA Lite application should show up. Then, login to KA Lite using the username and password you created during the installation process.
+    #. Copy and paste the URL into a web browser. The KA Lite application should show up.
+    #. Login to KA Lite using the username and password you created during the installation process.
+
+    * If you have forgotten the username/password combination, simply run ``kalite manage createsuperuser`` and follow the instructions.
```
Comment (aronasorman) [inline docs/usermanual/userman_admin.rst]: > simply run ``kalite manage createsuperuser`` in your terminal

File: docs/usermanual/userman_admin.rst [hunk 8]
```diff
@@ -29,57 +29,59 @@ If you choose to register online, these important terms will help you better und
...
 
 Registering Your Device with the Hub
 ____________________________________
 
 By registering your device with FLE, you can sync data back with our central data hub. This is useful for many reasons:
 
-    #. a project administrator can manage user accounts and view usage data from afar, without needing to physically visit the offline device.
-    #. usage data syncing back can inform the FLE team of multiple users in a certain geographic region, and we can connect people that might be able to help one another
-    #. it helps FLE and our partners understand where and how the software is being used, so we can keep adding features that support you!
+    #. Project administrator can manage user accounts and view usage data from afar, without the need to physically visit the offline device.
```
Comment (aronasorman) [inline docs/usermanual/userman_admin.rst]: > The project administrator

File: docs/usermanual/userman_admin.rst [hunk 9]
```diff
@@ -427,13 +430,13 @@ _____________________
 
 Allowing Other Users to Connect
 --------------------------------
-In order for other users to be able to connect to the KA Lite server with a different machine, you will need to give them an IP address with which to access the software. This should be different from the IP address given to you during setup (127.0.0.1:8008).
+In order for other users to be able to connect with the KA Lite from different computers, you will need to give them an IP address with which to access the software. The rest of the users should use the second IP address, different from the one you use (usually 127.0.0.1:8008 received during setup).
```
Comment (aronasorman) [inline docs/usermanual/userman_admin.rst]: > to connect with KA Lite

File: docs/usermanual/userman_admin.rst [hunk 10]
```diff
@@ -460,28 +463,28 @@ After registering your device:
...
+Save the videos in the ``CONTENT_ROOT`` directory of your installation. By default, that
 is the ``.kalite/content/`` folder in the *home directory* of the user running KA Lite.
 
 On Windows, navigate to something like ``C:\Documents and Settings\<username>\.kalite\content``.
 
-.. note:: If the drive where your ``.kalite/`` folder does not have enough free disk space,
-          you can change your `Configuration Settings`_, the one named ``CONTENT_ROOT`` and
-          define a folder where your videos are downloaded to. Remember that you may want
-          to move the contents of your old ``content/`` folder.
+.. note:: If the drive where your ``.kalite/`` folder is located does not have enough free disk space,
```
Comment (aronasorman) [inline docs/usermanual/userman_admin.rst]: > Remember to move the contents

File: docs/usermanual/userman_admin.rst [hunk 11]
```diff
@@ -493,22 +496,15 @@ When you have copied in the files, to register them with KA Lite, follow the fol
...
-
-If you skipped the step for adding assessment items during install, run the following command (beware it can take a while, the download size is over 500 MB):
-
-``kalite manage unpack_assessment_zip https://learningequality.org/downloads/ka-lite/0.15/content/khan_assessment.zip -f``
-
-As an alternative, you can download the zip file from https://learningequality.org/downloads/ka-lite/0.15/content/khan_assessment.zip and run the command on your local copy. This is useful if you're deploying KA Lite on multiple computers.
+Adding Languages
+----------------
 
+KA Lite content is offered in several languages. If your language is available you can download the **contentpack** for it with all the available subtitles and user interface translations. After you download and install the **contentpack** for a desired language, KA Lite will give you the option to download individual dubbed videos from that language Khan Academy YouTube channel. For more technical background about the new **contentpacks**, please refer to our `Wiki page <https://github.com/learningequality/ka-lite/wiki/Content-packs>`_.
```
Comment (aronasorman) [inline docs/usermanual/userman_admin.rst]: > from that language's Khan Academy YouTube channel

**Comment by aronasorman** [comment]
Minor corrections.

File: docs/faq.rst (hunk 4 above)
Comment (radinamatic) [inline docs/faq.rst]: I wouldn't be me if I didn't eat an article or two... :stuck_out_tongue_winking_eye:

File: docs/usermanual/userman_admin.rst (hunk 8 above)
Comment (radinamatic) [inline docs/usermanual/userman_admin.rst]: I must have been very hungry last night... :unamused:

File: docs/usermanual/userman_admin.rst (hunk 10 above)
Comment (radinamatic) [inline docs/usermanual/userman_admin.rst]: This reminder in is in the second sentence of the note. 
@aronasorman

Comment (aronasorman) [inline docs/usermanual/userman_admin.rst]: Yup! My main change here is removing the `also` from that sentence.

**Comment by aronasorman** [comment]
Regarding the test failures: you forked from a non-working commit. Should be fixed again once we merge this to 0.16.x.

**Pull request merged**